pkg/synopsysctl/cmd_delete.go

```diff
@@ -190,6 +194,15 @@ var deletePolarisCmd = &cobra.Command{
 			}
 		}
 
+		// Delete StatefulSet PVCs
+		if list, err := kubeClient.CoreV1().PersistentVolumeClaims(namespace).List(metav1.ListOptions{LabelSelector: fmt.Sprintf("environment=%s", p.Namespace)}); err == nil {
+			for _, v := range list.Items {
+				if err := kubeClient.CoreV1().PersistentVolumeClaims(namespace).Delete(v.Name, &metav1.DeleteOptions{}); err != nil {
```

Comment:
Whether we need to support this use case?

Reply:
If so, can we provide some warning if the reclaim policy is `delete`?

Reply:
As discussed, there is now a warning and the user is asked whether the PVC should be deleted or not.